.travis.yml

```diff
@@ -5,28 +5,33 @@ os:
   - linux
   - osx
 
+git:
+  depth: 10
+  submodules: false
```

Comment:
If we ever did start using git submodules (perish the thought), would this be here waiting to break the build?  Or would it Just Work™?

Reply:
This is preemptive punishment for using git submodules. I fully expect Travis to catch fire, Github to go down, locusts to eat our crops and all manner of bad things to happen.

(Obsolete now.)